compiler/arm/codegen/OMRTreeEvaluatorTable.hpp

```diff
@@ -779,7 +779,11 @@
    TR::TreeEvaluator::exceptionRangeFenceEvaluator,       // TR::exceptionRangeFence
    TR::TreeEvaluator::exceptionRangeFenceEvaluator,       // TR::dbgFence
    TR::TreeEvaluator::NULLCHKEvaluator,     // TR::NULLCHK
+#if J9_PROJECT_SPECIFIC
    TR::TreeEvaluator::resolveCHKEvaluator,  // TR::ResolveCHK
+#else
+   TR::TreeEvaluator::unImpOpEvaluator,     // TR::ResolveCHK
+#endif
```

Comment:
Just add `TR::TreeEvaluator::unImpOpEvaluator` unconditionally for `TR::ResolveCHK` in OMR.  Downstream projects (like OpenJ9) will modify this table entry as appropriate if they provide an implementation.

Reply:
Fixed.